### ISSUE TITLE
Suggestion: Seed extractor (fruits into seeds)

### ISSUE THREAD

**Issue by Slayerhero90** [naelstrof/KoboldKare#142]
It's not the end of the world to have to run into town to buy more seeds when your farm runs out of fruit, but it's certainly inconvenient. Perhaps, towards the late teens or so of Star progression, a seed extractor could be built which takes fruit and yields seed packets at a 1:1 ration.

A bonus of this is that it would allow you incorporate more types of fruit you can find in the wild without having to set up a shop for them. You could make bananas farmable, for instance (careful to use something other than water to water them!), or it could be a place to put more unusual size boosting fruit like, say, something that makes the tail longer.

**Comment by naelstrof** [comment]
Yeah its a good idea. And in fact bananas have custom banana tree plants and seeds already made. There was a seed extractor in the past, but it was cut when a bunch of systems changed and I couldn't keep it maintained. It may or may not come back later.
